### ISSUE TITLE
Node Management: Farmer can configure the public bridge

### ISSUE THREAD

**Issue by muhamadazmy** [threefoldtech/tfgrid-sdk-ts#114]

**Comment by xmonader** [comment]
Is this still needed ?  @coesensbert @muhamadazmy

**Comment by coesensbert** [comment]
> Is this still needed ? @coesensbert @muhamadazmy

yes please

**Comment by alichaddad** [comment]
To get this done we will need to have access to the user's mnemonics to sign the request and with the current integration using polkadot this won't make for the best user experience, since the user will have to do that for each request which doesn't make sense when he is simply checking the node configuration. There was an issue to use the profile manager in dashboard <https://github.com/threefoldtech/tfgrid-sdk-ts/issues/470> which should fix this and work can't start here until this is done.

**Comment by muhamadazmy** [comment]
Actually I am not sure if we need this anymore. While the farmer can still actually set the exit interface of the node, we don't persist this on the node anymore. The persistence of which exit interface is used caused other issues, instead the farmer has to have his network well configured for ipv6 and let zos auto-detection take place.

We still can show which setup the node has, similar to what we show on zos console. But over the web ? and may be only if you click a button because this required an `rmb` call to the node to query the current node setup.

So basically same as above BUT with no ability to set it up

**Comment by xmonader** [comment]
Have to push that to next, with the current prios it won't be done in 3.11
